### PR TITLE
Add Intl Utils

### PR DESCRIPTION
**Pull request by imGok**
Since `flutter pub run intl_translation:generate_from_arb` doesn't generate files with null safety, I suggest to add a recent package that support null safety : https://pub.dev/packages/intl_utils 

command : `flutter pub run intl_utils:generate`

We can discuss about output file tho

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -181,3 +182,23 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+
```
Comment (hoangdat) [inline pubspec.yaml:185]: it a big change, please put an ADR to clear about that. Thanks

**Comment by huynguyennovem** [comment]
There is another simpler solution for this:

Add this in the end of `pubspec.yaml` file:
```
flutter_intl:
  enabled: true
```

**Comment by hoangdat** [comment]
@imGok any update for this.

**Comment by imGok** [comment]
@hoangdat I still want to implement this because the file crashed all the time

File: pubspec.yaml (hunk 1 above)
Comment (hoangdat) [reply pubspec.yaml:185]: @imGok please describe your solution in ADR.

**Comment by hoangdat** [comment]
@imGok @huynguyennovem IMO, we don't need this fix.

**Comment by imGok** [comment]
Fixed by adding intl_generator

**Pull request closed without merging**